barretenberg/cpp/src/barretenberg/vm/avm/trace/trace.cpp

```diff
@@ -2235,26 +2216,32 @@ void AvmTraceBuilder::op_sload(uint8_t indirect, uint32_t slot_offset, uint32_t
     auto clk = static_cast<uint32_t>(main_trace.size()) + 1;
 
     auto [resolved_slot, resolved_dest] = unpack_indirects<2>(indirect, { slot_offset, dest_offset });
-    auto read_slot = constrained_read_from_memory(
-        call_ptr, clk, resolved_slot, AvmMemoryTag::FF, AvmMemoryTag::U0, IntermRegister::IA);
 
+    auto read_slot = unconstrained_read_from_memory(resolved_slot);
```

Comment:
Can you please open a ticket about creating a specific gadget for ssload and sstore and reference it here? Otherwise, it is not in the pipeline of the remaining work.